spec/models/manageiq/providers/openshift/container_manager/container_template_spec.rb

```diff
@@ -0,0 +1,61 @@
+describe ContainerTemplate do
+  let(:ems) do
+    hostname = 'host.example.com'
+    token = 'theToken'
+    FactoryGirl.create(
+      :ems_openshift,
+      :name                      => 'OpenShiftProvider',
+      :connection_configurations => [{:endpoint       => {:role     => :default,
+                                                          :hostname => hostname,
+                                                          :port     => "8443"},
+                                      :authentication => {:role     => :bearer,
+                                                          :auth_key => token,
+                                                          :userid   => "_"}},
+                                     {:endpoint       => {:role     => :hawkular,
+                                                          :hostname => hostname,
+                                                          :port     => "443"},
+                                      :authentication => {:role     => :hawkular,
+                                                          :auth_key => token,
+                                                          :userid   => "_"}}]
+    )
+  end
+
+  before(:each) do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
+  end
+
+  it "instantiate a template with parameters and object labels" do
+    param = FactoryGirl.create(:container_template_parameter,
+                               :name     => 'VAR',
+                               :value    => 'example',
+                               :required => true)
+
+    object = {:apiVersion => "v1",
+              :kind       => "PersistentVolumeClaim",
+              :metadata   => {:name => "pvc-${VAR}"},
+              :spec       => {:accessModes => ["ReadWriteOnce"],
+                              :resources   => {:requests => {:storage => "8Gi"}}}}
+
+    object_labels = {:created_from_template => "true"}
+
+    template = FactoryGirl.create(:container_template,
+                                  :ems_id        => ems.id,
+                                  :objects       => [object],
+                                  :object_labels => object_labels).tap do |temp|
```

Comment:
@zakiva travis is complaining about this line:
`undefined method `object_labels=' for #<ContainerTemplate:0xXXX>`